config/locales/hello.pl.yml

```diff
@@ -0,0 +1,68 @@
+# https://github.com/hello-gem/hello
+
+pl:
+  hello:
+    locale_name: "Polski"
+
+    mailer:
+      welcome:
+        subject: "Witamy na naszej stronie"
+      confirm_email:
+        subject: "Potwierdz e-mail"
+      forgot_password:
+        subject: "Instrukcje resetowania hasła"
+
+    entities:
+      sign_out:
+        success: "Zostałeś(aś) wylogowany!"
+      unlink_access:
+        success: "Odłączono urządzenie od twojego konta"
+        alert: "Nie udało się odłaczyć urządzenia"
+      session_switch:
+        success: "Pomyślnie przełączono konta!"
+      session_forget:
+        success: "Zapomniana"
+      sign_up:
+        success: "Pomyślnie się zarejestrowany(aś)"
```

Comment:
Leftover from a previous translation (zarejestrowany). Would something more general be suitable here, to avoid the parentheses for feminine form? Like "Rejestracja przebiegła pomyślnie" or "Rejestracja pomyślna"? Same goes for line 17. It's just a suggestion.